src/shared/components/ResourceEditor/index.tsx

```diff
@@ -261,7 +261,11 @@ const ResourceEditor: React.FunctionComponent<ResourceEditorProps> = props => {
                   style={switchMarginRight}
                 />
               )}
-              {userAuthenticated && (
+              <AccessControl
+                path={[`${orgLabel}/${projectLabel}`]}
+                permissions={['resources/write']}
```

Comment:
Would be nice to have these permissions as an enum and more typesafety somewhere to avoid chances of typos because right now if I pass `['abc']` to permissions, I don't get any errors.

Reply:
the component is pre-built in the SDK, will do that when we got the SDK part (🤷🏻)